### PR TITLE
currently selected comic style test

### PR DESCRIPTION
**Pull request by polygirl**
fixes issue #166

### CODE REVIEW

File: e2e/src/comics.e2e-spec.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ describe('Comics', () => {
   it(
     'should create a comic thumbnail with the correct value\n' +
     'should open the expanded panel with the correct title\n' +
-    'should use the arrow button to go to the next comic',
+    'should use the arrow button to go to the next comic\n',
```
Comment (SubJunk) [inline e2e/src/comics.e2e-spec.ts]: This line can be reverted

File: e2e/src/comics.e2e-spec.ts [hunk 2]
```diff
@@ -30,6 +30,12 @@ describe('Comics', () => {
     );
   });
 
+  it('should use correct style for currently selected comic', async () => {
+      expect(
```
Comment (SubJunk) [inline e2e/src/comics.e2e-spec.ts]: Wrong indentation on this line

File: e2e/src/comics.e2e-spec.ts [hunk 3]
```diff
@@ -30,6 +30,12 @@ describe('Comics', () => {
...
+      await element(by.css('img.responsive-img.current')).isDisplayed()
```
Comment (SubJunk) [inline e2e/src/comics.e2e-spec.ts]: `isDisplayed` is already a statement so I don't think we need it to be wrapped in an `expect` here. Just have this line on its own

Comment (polygirl) [reply e2e/src/comics.e2e-spec.ts]: Happy to do this.  I followed the same pattern for collections spec on line 28 where I did the single line, and you asked for it to have an assertion.  Can this be different for a reason?

**Pull request merged**